#! /bin/sh /usr/share/dpatch/dpatch-run
## app_sql_postgres.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
##   30/6/05 Mark Purcell <msp@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Setup for postgresql under Debian (autoconf would be nicer)

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./apps/Makefile /tmp/dpep-work.ZD5tj4/asterisk-1.0.9.dfsg/apps/Makefile
--- ./apps/Makefile	2005-07-04 01:09:46.199583056 +0100
+++ /tmp/dpep-work.ZD5tj4/asterisk-1.0.9.dfsg/apps/Makefile	2005-07-04 01:11:03.511484541 +0100
@@ -43,6 +43,17 @@
 APPS+=$(shell if [ -f /usr/local/include/zaptel.h ]; then echo "app_zapras.so app_meetme.so app_flash.so app_zapbarge.so app_zapscan.so" ; fi)
 APPS+=$(shell if [ -f /usr/include/osp/osp.h ]; then echo "app_osplookup.so" ; fi)
 
+#
+## PGSQL stuff...  Autoconf anyone??
+#
+CFLAGS+=$(shell if [ -d /usr/local/pgsql/include ]; then echo "-I/usr/local/pgsql/include"; fi)
+CFLAGS+=$(shell if [ -d /usr/include/pgsql ]; then echo "-I/usr/include/pgsql"; fi)
+CFLAGS+=$(shell if [ -d /usr/include/postgresql ]; then echo "-I/usr/include/postgresql"; fi)
+CFLAGS+=$(shell if [ -d /usr/local/include/pgsql ]; then echo "-I/usr/local/include/pgsql"; fi)
+CFLAGS+=$(shell if [ -d /opt/pgsql/include ]; then echo "-I/opt/pgsql/include"; fi)
+#CFLAGS+=$(shell if [ -f /usr/include/libpq-fe.h ]; then echo "-I/usr/include"; fi)
+CFLAGS+=$(shell if [ -f /usr/include/postgresql/8.0/libpq-fe.h ]; then echo "-I/usr/include/postgresql/8.0"; fi)
+
 CFLAGS+=-fPIC -DPIC
 
 ifeq ($(USE_POSTGRES_VM_INTERFACE),1)
diff -urNad --exclude=CVS --exclude=.svn ./cdr/Makefile /tmp/dpep-work.ZD5tj4/asterisk-1.0.9.dfsg/cdr/Makefile
--- ./cdr/Makefile	2005-07-04 01:09:46.200582848 +0100
+++ /tmp/dpep-work.ZD5tj4/asterisk-1.0.9.dfsg/cdr/Makefile	2005-07-04 01:09:49.153967870 +0100
@@ -56,6 +56,7 @@
 CFLAGS+=$(shell if [ -d /usr/local/include/pgsql ]; then echo "-I/usr/local/include/pgsql"; fi)
 CFLAGS+=$(shell if [ -d /opt/pgsql/include ]; then echo "-I/opt/pgsql/include"; fi)
 #CFLAGS+=$(shell if [ -f /usr/include/libpq-fe.h ]; then echo "-I/usr/include"; fi)
+CFLAGS+=$(shell if [ -f /usr/include/postgresql/8.0/libpq-fe.h ]; then echo "-I/usr/include/postgresql/8.0"; fi)
 MLFLAGS=
 MLFLAGS+=$(shell if [ -d /usr/lib/pgsql ]; then echo "-L/usr/lib/pgsql"; fi)
 MLFLAGS+=$(shell if [ -d /usr/local/pgsql/lib ]; then echo "-L/usr/local/pgsql/lib"; fi)
diff -urNad asterisk-1.0.3/cdr/Makefile /tmp/dpep.FJ3Q8L/asterisk-1.0.3/cdr/Makefile
--- asterisk-1.0.3/cdr/Makefile 2005-01-15 18:28:29.000000000 +0100
+++ /tmp/dpep.FJ3Q8L/asterisk-1.0.3/cdr/Makefile        2005-01-15 18:28:29.000000000 +0100
@@ -12,7 +12,7 @@
 #

 #ADD cdr_pgsql.so to MODS= to include PostgreSQL support: REQUIRES PostgreSQL libs
-MODS=cdr_csv.so cdr_manager.so
+MODS=cdr_csv.so cdr_manager.so cdr_pgsql.so


 CFLAGS+=-fPIC -DPIC
:
